examples/plotting/wcsaxes_plotting_example.py

```diff
@@ -0,0 +1,54 @@
+"""
+===========================
+WCS Axes Plotting
+===========================
```

Comment:
```suggestion
================
WCSAxes plotting
================
```

Reply:
oh! so many mistakes!  my apologies. I will be more conscious next time.